projects/ion/src/lib/popover/popover.directive.spec.ts

```diff
@@ -234,6 +234,12 @@ describe('Popover host tests', () => {
       expect(directive.closePopover).toHaveBeenCalled();
     }
   );
+
+  it('should not open new popover when be opened', () => {
```

Comment:
```suggestion
  it('should not open a new popover when a popover is already opened', () => {
```
This will improve readability.

Reply:
thanks for the tip